packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -12,7 +12,88 @@
 }
 
 .wp-block-navigation-menu-item {
-	.wp-block-navigation-menu-item {
+	.wp-block-navigation-menu-link {
 		margin-left: $grid-size-large;
 	}
 }
+
+/**
+ * Colors Selector component
+ */
+$colors-selector-size: 22px;
+.block-editor-block-colors-selector {
+	width: auto;
+
+	// Toolbar colors-selector button.
+	.block-editor-block-colors-selector__toggle {
+		display: block;
+		margin: 0 auto;
+		padding: 3px;
+		width: auto;
+
+		// Button container.
+		.block-editor-block-colors-selector__icon-container {
+			width: 42px;
+			height: 30px;
+			position: relative;
+			margin: 0 auto;
+			padding: 3px;
+			display: flex;
+			align-items: center;
+			border-radius: 4px;
+
+			// colors-selector - selection status.
+			.colors-selector__state-selection {
+				font-size: 11px;
+				font-style: normal;
+				font-family: inherit;
+				font-weight: 600;
+
+				border-radius: $colors-selector-size / 2;
+				box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.2);
+
+				width: $colors-selector-size;
+				min-width: $colors-selector-size;
+				height: $colors-selector-size;
+				min-height: $colors-selector-size;
+				line-height: ($colors-selector-size - 2);
+			}
+
+			// Add the button arrow.
+			&::after {
+				@include dropdown-arrow();
+			}
+
+			// Styling button states.
+			&:focus,
+			&:hover {
+				color: #555d66;
+				box-shadow: inset 0 0 0 1px #555d66, inset 0 0 0 2px #fff;
```

Comment:
these colors should come from the global color list: https://github.com/WordPress/gutenberg/blob/c2c8276ac8e89af8382d5180ffa6a52f7fa3f98e/assets/stylesheets/_colors.scss#L11